pkg/controller/installation/core_controller.go

```diff
@@ -1226,14 +1242,15 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 	components = append(components,
 		rcertificatemanagement.CertificateManagement(&rcertificatemanagement.Config{
 			Namespace:       common.CalicoNamespace,
-			ServiceAccounts: []string{render.CalicoNodeObjectName, render.TyphaServiceAccountName},
+			ServiceAccounts: []string{render.CalicoNodeObjectName, render.TyphaServiceAccountName, kubecontrollers.KubeControllerMetrics},
```

Comment:
These should be service accounts. Please remove that value and add these two: [kubecontrollers.KubeControllerServiceAccount, kubecontrollers.EsKubeControllerServiceAccount]